src/App.js

```diff
@@ -1,19 +1,56 @@
 import React from 'react';
-import './App.css';
 
+import TodoList from './components/TodoList/TodoList';
+import NewTodo from './components/NewTodo/NewTodo';
 import users from './api/users';
+import todos from './api/todos';
+import './App.css';
+
+class App extends React.Component {
+  state = {
+    todos,
+    userTodos: [],
+  }
+
+  handleChangeTodos = (id) => {
+    this.setState(state => ({
+      userTodos: state.todos.filter(todo => todo.userId === id),
+    }));
+  }
+
+  handleSubmitTodo = (userId, title) => {
+    this.setState(state => ({
+      todos: [
+        ...state.todos,
+        {
+          userId,
+          id: state.todos.length + 1,
+          title,
+          completed: false,
+        },
+      ],
+    }));
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <h1 className="App__header">Add todo form</h1>
+
+        <p className="App__text">
+          <span className="App__span">Users: </span>
+          {users.length}
+        </p>
 
-function App() {
-  return (
-    <div className="App">
-      <h1>Add todo form</h1>
-
-      <p>
-        <span>Users: </span>
-        {users.length}
-      </p>
-    </div>
-  );
+        {<TodoList todos={this.state.userTodos} />}
+        {<NewTodo
+          users={users}
+          handleChange={this.handleChangeTodos}
+          handleSubmit={this.handleSubmitTodo}
+        />}
```

Comment:
Ти записуєш в стейті в "todos" зміни, але їх не виводиш.
Підозрюю, помилка друкарська, т.я. якщо його дрібку переписати, усе почне виводитись:
```suggestion
        {<NewTodo
          users={users}
          handleChange={this.handleChangeTodos}
          handleSubmit={this.handleSubmitTodo}
        />}

        {<TodoList todos={this.state.todos} />}
```